pages/_app.js

```diff
@@ -1,10 +1,20 @@
 import GlobalStyle from "../styles";
+import Layout from "@/components/Layout";
+import { SWRConfig } from "swr";
+
+const fetcher = (url) => fetch(url).then((response) => response.json());
 
 export default function App({ Component, pageProps }) {
+  const userId = "65e0925792f086ae06d2eadb";
+
   return (
     <>
-      <GlobalStyle />
-      <Component {...pageProps} />
+      <Layout>
+        <GlobalStyle />
+        <SWRConfig value={{ fetcher }}>
+          <Component {...pageProps} userId={userId} />
```

Comment:
where is the userID being used?

If it is not used anywhere, remove this prop and also the variable in line 8